lib/enum/basic_enum.rb

```diff
@@ -85,66 +97,78 @@ def element_frequency_count
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-find-all
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-select
   def elements_ending_in_er
+    collection.select{ |word| word.end_with?('er')}
   end
 
 
   # Refer to find_index
   # http://ruby-doc.org/core-2.1.4/Enumerable.html#method-i-find-index
   def index_of_first_awesome_element
+    collection.find_index{ |hash| hash[:awesome] == true }
```

Comment:
Since `hash[:awesome]` is already truthy, you don't need to check `== true`. Also, use `|person|` or another intuitive name instead of `|hash|`.